specification/dartLangSpec.tex

```diff
@@ -19072,23 +19072,27 @@ \section*{Appendix: Integer Implementations}
 IEEE-754 64-bit floating point number with no fractional part.
 This includes positive and negative \Index{infinity},
 which can be reached by overflowing
-(integer division by zero is still not allowed).
+(integer division by zero is still a dynamic error).
 Otherwise valid integer literals (including any leading minus sign)
 that represent invalid JavaScript \code{int} values
-cannot be compiled to JavaScript.
-Operations on integers may lose precision since 64-bit floating point numbers
-are limited to 53 significant bits.
+are compile-time errors when compiling to JavaScript.
+Operations on integers may lose precision,
+because they are represented as 64-bit floating point numbers
```

Comment:
> ... because the result is represented as a 64-bit floating point number,
> which are limited to 53 significant bits.

The "they" is unclear, it can refer both to integers and operations, and it makes sense for neither since it is the result which may lose precision.

Reply:
Clarified the wording: Both the operands and the result are represented as mentioned, which implies that precision may be lost (for instance, when the result is an odd integer >=2^53).